cdtoolchainv2/cd_toolchain_v2.go

```diff
@@ -123,6 +123,7 @@ func GetServiceURLForRegion(region string) (string, error) {
 		"jp-tok": "https://api.jp-tok.devops.cloud.ibm.com/toolchain/v2", // The toolchain API endpoint in the jp-tok region
 		"au-syd": "https://api.au-syd.devops.cloud.ibm.com/toolchain/v2", // The toolchain API endpoint in the au-syd region
 		"ca-tor": "https://api.ca-tor.devops.cloud.ibm.com/toolchain/v2", // The toolchain API endpoint in the ca-tor region
+		"ca-mon": "https://api.ca-mon.devops.cloud.ibm.com/toolchain/v2", // The toolchain API endpoint in the ca-mon region
```

Comment:
Mike Rennie said we should make it clear that this is a limited availability region

```suggestion
		"ca-mon": "https://api.ca-mon.devops.cloud.ibm.com/toolchain/v2", // Montreal (ca-mon) is a limited-availability region and not generally available. The host URL for Tekton Pipeline Service in the ca-mon region.
```